src/server.js

```diff
@@ -521,7 +521,7 @@ function getClientConfiguration() {
 	config.themes = themes.getAll();
 
 	if (config.displayNetwork) {
-		config.defaults = Helper.config.defaults;
+		config.defaults = _.clone(Helper.config.defaults);
```

Comment:
We should not call that defaults anymore IMO. Not necessarily for your PR though.